stable/mongodb-replicaset/templates/mongodb-statefulset.yaml

```diff
@@ -34,7 +34,8 @@ spec:
 {{ toYaml .Values.securityContext | indent 8 }}
       initContainers:
         - name: copy-config
-          image: busybox
+          image: "{{ .Values.copyConfigImage.repository }}"
```

Comment:
Yep, but i no have a time to pushing tihis change.
Since i wrote the admission hook that replace  address with our private repo i don't need to rewrite a lot of charts to fix such things.